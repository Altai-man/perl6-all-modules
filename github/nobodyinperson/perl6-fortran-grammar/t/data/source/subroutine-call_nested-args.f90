call sub(1,2.34,func(),var,var2(1_wp,:))
