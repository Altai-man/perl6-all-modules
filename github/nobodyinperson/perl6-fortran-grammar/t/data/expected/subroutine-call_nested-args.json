{
  "name": "sub",
  "arguments": {
    "argument": [
      {
        "value-returning-code": {
          "in-place": {
            "atomic": {
              "number": {
                "integer": {
                  "digits": "1"
                }
              }
            }
          }
        }
      },
      {
        "value-returning-code": {
          "in-place": {
            "atomic": {
              "number": {
                "float": {
                  "digits": [
                    "2",
                    "34"
                  ]
                }
              }
            }
          }
        }
      },
      {
        "value-returning-code": {
          "function-call": {
            "name": "func"
          }
        }
      },
      {
        "value-returning-code": {
          "accessed-variable": {
            "name": "var"
          }
        }
      },
      {
        "value-returning-code": {
          "accessed-variable": {
            "indexed-array": {
              "name": "var2",
              "array-indices": {
                "array-index": [
                  {
                    "integer": {
                      "digits": "1",
                      "precision-spec": {
                        "name": "wp"
                      }
                    }
                  },
                  {
                    "array-index-region": {
                      "value-returning-code": [
                        
                      ]
                    }
                  }
                ]
              }
            }
          }
        }
      }
    ]
  }
}
